library/Vanilla/Models/SSOInfo.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * @author Alexandre (DaazKu) Chouinard <alexandre.c@vanillaforums.com>
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+namespace Vanilla\Models;
+
+use Vanilla\ArrayAccessTrait;
+
+/**
+ * Class SSOInfo
+ *
+ * This is a data object containing information returned by an SSOAuthenticator.
+ * Adding methods to this object probably means that you are doing something wrong.
+ *
+ * @package Vanilla\Models
+ */
+class SSOInfo implements \ArrayAccess {
+    use ArrayAccessTrait;
+
+    /**
+     * Maps to "GDN_UserAuthenticationProvider.AuthenticationSchemeAlias"
+     *
+     * @var string
```

Comment:
If it's possible to fit a variable's doc on a single line, you should stick with the one-line syntax.

```php
/** @var string Maps to "GDN_UserAuthenticationProvider.AuthenticationSchemeAlias" */
```